### PR TITLE
Refactor(web-react): No longer use default props in the rest of the c…

### PR DESCRIPTION
**Pull request by crishpeen**
…omponents

### CODE REVIEW

File: packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx [hunk 1]
```diff
@@ -12,7 +12,13 @@ type AttachmentImagePreviewProps = {
...
 };
 
-const AttachmentImagePreview = ({ label, imagePreview, meta, imageObjectFit }: AttachmentImagePreviewProps) => {
+const defaultProps: Partial<AttachmentImagePreviewProps> = {
+  meta: undefined,
+  imageObjectFit: 'cover',
+};
+
+const AttachmentImagePreview = (props: AttachmentImagePreviewProps) => {
+  const { label, imagePreview, meta = defaultProps.meta, imageObjectFit = defaultProps.imageObjectFit } = props;
```
Comment (pavelklibani) [inline packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx:21]: why is it here like this and not like in other components?

```const propsWithDefaults = { ...defaultProps, ...props };```

Comment (crishpeen) [reply packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx:21]: because TS hates me 🤷

Comment (pavelklibani) [reply packages/web-react/src/components/FileUploader/AttachmentImagePreview.tsx:21]: i think you need to move `AttachmentImagePreviewProps` to types file and then it should be working

**Review by pavelklibani — APPROVED** [review]

**Review by literat — APPROVED** [review]
LGTM 🦖

**Pull request merged**